db/migrate/20230807040058_move_web_hooks_to_new_event_ids.rb

```diff
@@ -0,0 +1,132 @@
+# frozen_string_literal: true
+
+class MoveWebHooksToNewEventIds < ActiveRecord::Migration[7.0]
+  def up
+    execute <<~SQL
+      INSERT INTO web_hook_event_types_hooks(web_hook_event_type_id, web_hook_id)
```

Comment:
Is the table name incorrect here? I don't have `web_hook_event_types_hooks` in my DB...

Reply:
Ah I found it  https://github.com/discourse/discourse/blob/f17b70ede33e44da5a5adf723d4fb312ce92a3ea/db/migrate/20160905085445_create_join_table_web_hooks_web_hook_event_types.rb , we should have a model file for this, can you make one please so it's more obvious?

So I guess what we are saying now is:

* Previously there was 1 entry in this table, `TOPIC` which is `web_hook_event_type_id` of `1`, which represented a wildcard
* After this migration there will be `5` records in this table, one for each of the `TYPES[:topic]` events per web hook, since those 5 make up the wildcard now?

Maybe we should just do a migration spec too to confirm expected outcome.

Reply:
Good idea about missing model file for many-to-many table 🚀 

Logic about number is a bit different.

Before `1` represented all topic events. And now we have 1 - topic created, 2 - topic updated etc. 

This migration is solving only individual events. 

Wildcard webhooks are untouched and they are represented by `WebHook.wildcard_web_hook` boolean table.

Logic to decide and webhook is active is here: https://github.com/discourse/discourse/blob/main/app/models/web_hook.rb#L46